src/test/java/io/candydoc/plugin/save_ubiquitous_language/file/json/UbiquitousLanguageAsJsonTest.java

```diff
@@ -45,6 +45,11 @@ public void save_bounded_context_as_json() {
     Path actualFile = DESTINATION_FOLDER.resolve("bounded_context_one.json");
 
     AssertFileContent.assertThatJson(actualFile).isEqualTo(referenceFile);
+
+    Path manifestReferenceFile = REFERENCE_FOLDER.resolve("MANIFEST_bounded_context_one.json");
```

Comment:
I'm not sure that the Manifest file should be test in each tests. I would make a nested test class for this.

```java
class UbiquitousLanguageAsJsonTest {
  // ... some tests ...

  @Nested
  class ManifestTest {
    @Test
    void generate_manifest_from_vocabulary_file_paths() {
      // ... GWT ...
    }
  }
}
```